src/inc/aduc/result.h

```diff
@@ -116,36 +116,47 @@ typedef enum tagADUC_FACILITY_UNKNOWN_Components
 
 typedef enum tagADUC_FACILITY_LOWERLAYER_Components
```

Comment:
are there any changes in this file, or is this just the result of running clang-format on this file?

Reply:
Result of running clang format. I need to address that bug.